javascript/index.js

```diff
@@ -31,35 +48,82 @@ function printMilliseconds() {
 }
 
 function printSplit() {
-  // ... your code goes here
+  let liElement = document.createElement('li');
+  liElement.innerHTML = chronometer.split();
+  splitsElement.appendChild(liElement);
 }
 
 function clearSplits() {
-  // ... your code goes here
+  splitsElement.innerHTML = '';
 }
 
 function setStopBtn() {
-  // ... your code goes here
+  chronometer.stop();
+  toggleBtn({
+    element: btnLeftElement,
+    classToRemove: 'stop',
+    classToToggle: 'start',
+    elementText: 'START'
+  });
+  toggleBtn({
+    element: btnRightElement,
+    classToRemove: 'split',
+    classToToggle: 'reset',
+    elementText: 'RESET'
+  });
 }
 
 function setSplitBtn() {
-  // ... your code goes here
+  chronometer.split();
+  printSplit();
 }
 
 function setStartBtn() {
-  // ... your code goes here
+  chronometer.start(printTime);
+  toggleBtn({
+    element: btnLeftElement,
+    classToRemove: 'start',
+    classToToggle: 'stop',
+    elementText: 'STOP'
+  });
+  toggleBtn({
+    element: btnRightElement,
+    classToRemove: 'reset',
+    classToToggle: 'split',
+    elementText: 'SPLIT'
+  });
 }
 
 function setResetBtn() {
-  // ... your code goes here
+  chronometer.reset();
+  clearSplits();
+  printTime(0);
 }
 
 // Start/Stop Button
+let toggleBtn = ({ element, classToRemove, classToToggle, elementText }) => {
+  element.classList.remove(classToRemove);
+  element.classList.toggle(classToToggle);
+  element.innerHTML = elementText;
+};
```

Comment:
Changing the button would have been easier, but this is also quite a good solution! If you are interested, you could take a look at the solution from the Student Portal.